apps/web/src/pages/settings/organization/OrganizationName.tsx

```diff
@@ -0,0 +1,64 @@
+import { Button, Input, successMessage } from '@novu/design-system';
+import { Controller, useForm } from 'react-hook-form';
+import { useRenameOrganization } from '../../../api/hooks';
+import { HStack } from '../../../styled-system/jsx';
+import { InputStyles, InputWrapperStyles } from './OrganizationName.styles';
+
+type FormValues = {
+  name: string;
+};
+
+export function OrganizationName({ organizationName }: { organizationName?: string }) {
+  const { isLoading, renameOrganization } = useRenameOrganization({
+    onSuccess: () => {
+      successMessage('Organization name updated');
+    },
+  });
+
+  const {
+    control,
+    formState: { isSubmitting, isDirty, isValid, errors },
+    handleSubmit,
+    reset,
+  } = useForm<FormValues>({
+    defaultValues: {
+      name: organizationName,
+    },
+  });
+
+  const onSubmit = async (data: FormValues) => {
+    renameOrganization(data);
+    reset(data);
+  };
+
+  return (
+    <form noValidate name="organization-name-form" onSubmit={handleSubmit(onSubmit)}>
+      <HStack gap={24} alignItems="flex-end">
```

Comment:
```suggestion
      <HStack gap={'150'} alignItems="flex-end">
```

Reply:
Sorry, I'll have to write something to make this clear to everyone since it's not immediately intuitive. Our design system is based around `'100' = 1 rem = 16px` for anything size related where the "100" is like a percentage. If you want a size of 24, you would use `'150'`. I need to update our Design System to properly prohibit putting in raw values -- it's the whole point of us defining the tokens.